### PR TITLE
blog nav in menu

### PR DESCRIPTION
**Pull request by merittrompeter**
Move blog link in main menu.

### CODE REVIEW

**Comment by henryyamamoto-concur** [comment]
Looks good to me. @nicklathe

File: _includes/top-navigation.html [hunk 1]
```diff
@@ -77,14 +77,23 @@
...
-                              <li><a href="{{ site.baseurl }}/blog">Blog</a></li>
                               <li><a href="{{ site.baseurl }}/tools-support/release-notes/index.html">Release Notes</a></li>
                               <li><a href="{{ site.baseurl }}/tools-support/reference/custom-list-items.html">Reference</a></li>
                               <li><a href="{{ site.baseurl }}/newsletter-request.html">Newsletter</a></li>
                             </ul>
                         </li>
                         <!-- End Tools & Support -->
-
+                        <!-- Blog  -->
+                        {% if page.url == '/blog' %}
```
Comment (nicklathe) [inline _includes/top-navigation.html]: For this to work, I think it needs to be /blog/ (with the trailing slash). At least that's how it needs to be for it to work locally on my machine.

**Comment by nicklathe** [comment]
Other than that one comment, this looks good. :shipit:

File: _includes/top-navigation.html (hunk 1 above)
Comment (nicklathe) [inline _includes/top-navigation.html]: Sorry, should have been more specific. Line 87 is where it needs to be /blog/ in order for the highlighting to work. At least that's the only way I could get it to work locally.

Comment (merittrompeter) [inline _includes/top-navigation.html]: No, no - that was me not reading clearly, thank you!

**Pull request merged**